elephant/spike_train_dissimilarity.py

```diff
@@ -363,6 +363,15 @@ def van_rossum_distance(spiketrains, time_constant=1.0 * pq.s, sort=True):
     for i, j in np.ndindex(k_dist.shape):
         vr_dist[i, j] = (
             k_dist[i, i] + k_dist[j, j] - k_dist[i, j] - k_dist[j, i])
+    
+    # Clip small negative values
+    if np.any(vr_dist < 0):
+        warnings.warn(
+                "van_rossum_distance: very small negative values encountered "
```

Comment:
Here, it would also be helpful to hint at possible causes. As we tested, this does not happen if the units are in ms, and the user could change the representation to prevent the warning if noticed. Maybe something like "... very small negative values encountered, setting them to zero. This is likely due to floating point error, which can happen if the spike times are small floating point values. In this case, a possible solution to prevent the warning is to change the time unit of the input spike trains for better precision (e.g., from seconds to milliseconds).".